ui-admin/src/portal/siteContent/HtmlPageEditView.tsx

```diff
@@ -49,16 +98,41 @@ const HtmlPageView = ({ htmlPage, readOnly, siteInvalid, setSiteInvalid, updateP
     { renderAddSectionButton(0) }
     {htmlPage.sections.map((section, index) => {
       return <div key={`${section.id}-${index}`} className="row g-0">
-        <div className="col-md-4 p-2">
-          <HtmlSectionEditor updatePage={updatePage} htmlPage={htmlPage} setSiteInvalid={setSiteInvalid}
-            siteInvalid={siteInvalid} section={section} sectionIndex={index} readOnly={readOnly}/>
+        <div className="col-md-4 px-2 pb-2">
+          <HtmlSectionEditor updateSection={updateSection(index)} setSiteInvalid={setSiteInvalid}
+            moveSection={moveSection(index)} removeSection={removeSection(index)} allowTypeChange={section.id === ''}
+            siteInvalid={siteInvalid} section={section} readOnly={readOnly}/>
         </div>
         <div className="col-md-8">
           <HtmlSectionView section={section}/>
         </div>
         { renderAddSectionButton(index + 1) }
       </div>
     })}
+    { footerSection && <div key={`${footerSection.id}`} className="row g-0">
```

Comment:
```suggestion
    { footerSection && <div className="row g-0">
```
I don't think we need a key on this div since it's not being rendered as part of an array